scripts/provenance_fake_data.py

```diff
@@ -353,66 +354,65 @@ def upload_fake_provanence_data(person_id=1, project_id=1):
                                         type="simulation_parameters",
                                     )
                                 ## add context concept as well ##
+                                try:
+                                    context = init_parameter_value.get(
+                                        "context", {}
+                                    ).get("property", None)
+                                    if context is not None:
+                                        print(f"adding concept context {context}")
+                                        add_concept(
+                                            concept=context,
+                                            object_id=parameter.get("id"),
+                                            type="simulation_parameters",
+                                        )
+                                except Exception as e:
```

Comment:
Universal exception. How are you expecting things here to fail? Are you expecting any of the lines to fail or one in particular? Could we be using `try..except..else` or `try...except...finally`?

Reply:
removed